timesys/timing-devices-connection.txt

```diff
@@ -11,431 +11,431 @@
+IA-13RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-13RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-13RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-13RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-13RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-13RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-13RaBPM:TI-AMCFPGAEVR:FMC2CH3    IA-13RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-14RaCtrl:CO-PSCtrl-BO:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC1CH2    IA-14RaCtrl:CO-PSCtrl-SI1:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC1CH3    IA-14RaCtrl:CO-PSCtrl-SI2:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC1CH4    IA-14RaCtrl:CO-PSCtrl-SI3:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-14RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-14RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-14RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-14RaBPM:TI-AMCFPGAEVR:FMC2CH3    IA-14RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-15RaCtrl:CO-PSCtrl-SI1:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC1CH2    IA-15RaCtrl:CO-PSCtrl-SI2:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC1CH3    IA-15RaCtrl:CO-PSCtrl-SI3:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC1CH4    IA-15RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-15RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-15RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-15RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-15RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-16RaCtrl:CO-PSCtrl-BO:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC1CH2    IA-16RaCtrl:CO-PSCtrl-SI1:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC1CH3    IA-16RaCtrl:CO-PSCtrl-SI2:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC1CH4    IA-16RaCtrl:CO-PSCtrl-SI3:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-16RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-16RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-16RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-16RaBPM:TI-AMCFPGAEVR:FMC2CH3    IA-16RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-17RaCtrl:CO-PSCtrl-BO:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC1CH2    IA-17RaCtrl:CO-PSCtrl-SI1:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC1CH3    IA-17RaCtrl:CO-PSCtrl-SI2:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC1CH4    IA-17RaCtrl:CO-PSCtrl-SI3:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-17RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-17RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-17RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-17RaBPM:TI-AMCFPGAEVR:FMC2CH3    IA-17RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-18RaCtrl:CO-PSCtrl-SI1:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC1CH2    IA-18RaCtrl:CO-PSCtrl-SI2:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC1CH3    IA-18RaCtrl:CO-PSCtrl-SI3:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC1CH4    IA-18RaCtrl:CO-PSCtrl-SI4:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC1CH5    IA-18RaCtrl:CO-PSCtrl-SI5:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC2CH1    IA-18RaCtrl:CO-PSCtrl-SI6:TIMIN
+IA-18RaBPM:TI-AMCFPGAEVR:FMC2CH2    IA-18RaCtrl:CO-PSCtrl-SI7:TIMIN
+IA-19RaBPM:TI-AMCFPGAEVR:FMC1CH1    IA-20RaCtrl:CO-PSCtrl-BO:TIMIN     
```

Comment:
@fernandohds564 , I tried to generate this file using <code>timesys</code> subpackage but a google python package was missing in my system. Patricia told me that this BBB is not in rack 19 but in rack 20. we need to check if this information is correct in the excel file as well...

Reply:
ok @xresende! I will check and make a commit to this PR.
Thanks!